### PR TITLE
chore: `meilisearch-http` readability improvements

### PR DESCRIPTION
**Pull request by ryanrussell**
## What does this PR do?
Readability improvements in `meilisearch-http`. 

Believe these are pretty straightforward; let me know if anything needs adjusted or reverted :)

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: meilisearch-http/tests/auth/payload.rs [hunk 1]
```diff
@@ -275,7 +275,7 @@ async fn error_api_key_empty_payload() {
 
 #[actix_rt::test]
 async fn error_api_key_malformed_payload() {
-    let content = r#"{"malormed": "payload""#;
```
Comment (Kerollmops) [inline meilisearch-http/tests/auth/payload.rs:278]: I don't understand how this test was passing...

**Review by Kerollmops — CHANGES_REQUESTED** [review]
Thank you for your PR, would it be possible to make the tests pass again? I will probably merge once they are passing!

File: meilisearch-http/tests/auth/payload.rs (hunk 1 above)
Comment (ryanrussell) [reply meilisearch-http/tests/auth/payload.rs:278]: NP. Thought that was potentially more a bug than a type, but didn't chase down where `content` is re-used...

Will look into what's going on with the failed test when I get cycles (hopefully tonight) and rebase.

Comment (ryanrussell) [reply meilisearch-http/tests/auth/payload.rs]: @Kerollmops I just rebased and removed this commit(it was isolated already). The rest are small docs changes, didn't anticipate a test breaking. 

Will put the `malformed` and test on the list for another day and PR :)

**Review by irevoire — APPROVED** [review]
Hey thank you @ryanrussell that’s perfect 🚀 
bors merge

**Review by Kerollmops — APPROVED** [review]
bors merge

**Comment by curquiza** [comment]
bors merge

**Pull request merged**